ngraph/core/include/ngraph/opsets/opset8_tbl.hpp

```diff
@@ -29,7 +29,7 @@ NGRAPH_OP(ConvolutionBackpropData, ngraph::op::v1)
 NGRAPH_OP(Cos, ngraph::op::v0)
 NGRAPH_OP(Cosh, ngraph::op::v0)
 NGRAPH_OP(CumSum, ngraph::op::v0)
-NGRAPH_OP(DeformableConvolution, ngraph::op::v1)
+NGRAPH_OP(DeformableConvolution, ngraph::op::v8)
```

Comment:
Please add new operations to the end of file.

Reply:
done